### PR TITLE
5 populate fooddetails data

### PR DESCRIPTION
**Pull request by tehnools**

### CODE REVIEW

File: tests/client/components/FoodDetails.test.js [hunk 1]
```diff
@@ -1,41 +1,79 @@
+// External Imports
```
Comment (jessica-morgan) [inline tests/client/components/FoodDetails.test.js]: Remove comment

File: tests/client/components/FoodDetails.test.js [hunk 2]
```diff
@@ -1,41 +1,79 @@
...
 import React from 'react'
+import { Provider } from 'react-redux'
+// Test Imports
```
Comment (jessica-morgan) [inline tests/client/components/FoodDetails.test.js]: remove comment

File: tests/client/components/FoodDetails.test.js [hunk 3]
```diff
@@ -1,41 +1,79 @@
...
+import { render, shallow } from 'enzyme'
+import configureStore from 'redux-mock-store'
+// Local Imports
```
Comment (jessica-morgan) [inline tests/client/components/FoodDetails.test.js]: remove comment

**Review by jessica-morgan — CHANGES_REQUESTED** [review]
There have been many changes to the package-lock file, please delete your package-lock file on your branch then pull from development branch, also please see comments in FoodDetails.test :)

File: tests/client/components/FoodDetails.test.js (hunk 1 above)
Comment (tehnools) [reply tests/client/components/FoodDetails.test.js]: Done

File: tests/client/components/FoodDetails.test.js (hunk 2 above)
Comment (tehnools) [reply tests/client/components/FoodDetails.test.js]: Done

Comment (tehnools) [reply tests/client/components/FoodDetails.test.js]: Done

File: tests/client/components/FoodDetails.test.js (hunk 3 above)
Comment (tehnools) [reply tests/client/components/FoodDetails.test.js]: Yeeted

**Review by jessica-morgan — APPROVED** [review]
Looks good, you can go ahead and merge with development branch

**Pull request merged**